### PR TITLE
Fix #235 skip checksum check for snapshot version

### LINKED ISSUE #235: Skip checksum check for SNAPSHOT version
In conf/uno.conf, if custom Accumulo build is defined/used by having ACCUMULO_REPO set to local repo, uno fetch command fetches the local accumulo repo build. After that any attempt to start accumulo gives following error

"ACCUMULO_HASH is not set. Set it for your version in 'conf/checksums' or uno.conf"

The issue is similar to the one fixed in fluo-muchos. 
https://github.com/apache/fluo-muchos/pull/262/

### PR DESCRIPTION
**Pull request by mandarinamdar**
Skip ACCUMULO_HASH check for Accumulo SNAPSHOT version
Allows uno to start accumulo for SNAPSHOT builds

### CODE REVIEW

File: bin/impl/load-env.sh [hunk 1]
```diff
@@ -139,7 +139,7 @@ if [[ -z "$ZOOKEEPER_HASH" ]]; then
   echo "ZOOKEEPER_HASH is not set. Set it for your version in 'conf/checksums' or uno.conf"
   exit 1
 fi
-if [[ -z "$ACCUMULO_HASH" ]]; then
+if [[ -z "$ACCUMULO_HASH" && "$ACCUMULO_VERSION" != *"SNAPSHOT"* ]]; then
```
Comment (arvindshmicrosoft) [inline bin/impl/load-env.sh]: I believe you don't need the double-quotes around $ACCUMULO_VERSION. On a similar note, I believe the double-quotes around SNAPSHOT are not needed as there are no embedded spaces in the "needle" string we are searching for. It is my understanding that the quotes would only be needed if there were embedded space(s) within the needle string.

Comment (mandarinamdar) [reply bin/impl/load-env.sh]: Thanks for the review @arvindshmicrosoft . Yes, right. I have included quotes mainly for consistency with the existing script

Comment (arvindshmicrosoft) [reply bin/impl/load-env.sh]: Fair enough.

**Comment by arvindshmicrosoft** [comment]
Thanks for the contribution @mandarinamdar! If you would like to be added as contributor, please [edit Fluo's people page](https://github.com/apache/fluo-website/edit/gh-pages/pages/people.md) and submit a PR.

The Apache Fluo project optionally [tweets](https://twitter.com/apachefluo) about first contributions. If you would like a tweet to be made about this PR, just let us know what your Twitter handle is.

**Comment by ctubbsii** [comment]
@mandarinamdar In addition to what @arvindshmicrosoft , please also consider subscribing to our developer mailing list (info at https://fluo.apache.org/contactus/), particularly if you intend to contribute regularly.

**Pull request merged**